.eslintrc.json

```diff
@@ -10,21 +10,28 @@
     "html"
   ],
   "rules": {
+    "padded-blocks": "off",
     "brace-style": "off",
-    "new-cap": ["error", { "capIsNewExceptions": ["Polymer"] }],
+    "new-cap": [
+      "error",
+      {
+        "capIsNewExceptions": ["Polymer", "ListItemUtils", "MissingOptions", "CommonFunctionality"],
```

Comment:
` "ListItemUtils", "MissingOptions", "CommonFunctionality"` this is specific to etools-dropdown... I don't think you need it here